pipelines/nextflow/conf/busco.config

```diff
@@ -24,13 +24,13 @@
 */
 
 params {
-        busco_version = 'v5.7.1_cv1' // metaeuk 'v5.4.7_cv1'
+        busco_version = 'v5.7.1_cv1'
```

Comment:
Do we want to update it to `5.8`? I believe you have tested it and it is working better than its predecessors.

Reply:
Already done, in between tasks right now but this will be applied yes.